cyclus_input/buildtimes/deployinst.xml

```diff
@@ -113,6 +114,7 @@
         <val>Comanche_Peak_2</val>
         <val>Watts_Bar_1</val></prototypes>
     <build_times>
+        <val>900</val>
```

Comment:
`caksed_spent_uox` is HLW,
while `tails` is LLW.
The prototype `Sink` is deployed in timestep 960, which is half accurate,
considering there are already LLW (or tails) repositories around.
I suggest creating a separate `sink` for LLW which is present from the beginning of the simulation.